factcast-spring-boot-autoconfigure/src/main/java/org/factcast/spring/boot/autoconfigure/redis/RedissonAutoConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2017-2020 factcast.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.spring.boot.autoconfigure.redis;
+
+import org.factcast.factus.utils.FactusDependency;
+import org.redisson.api.RedissonClient;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.Bean;
+
+@AutoConfiguration
+@ConditionalOnClass({RedissonClient.class})
+public class RedissonAutoConfiguration {
+
+  @Bean
```

Comment:
Technically `ConditionalOnClass` does not mean that a bean was created right?
I wonder if we should add a `ConditionalOnBean` to the bean providing method

Reply:
True, good point.